com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -270,8 +271,7 @@ void BindMotionVectorPassColorBuffers(in RenderGraphBuilder builder, in PrepassO
                 // NOTE: Currently we profiled that generating the HTile for SSR and using it is not worth it the optimization.
                 // However if the generated HTile will be used for something else but SSR, this should be made NOT resolve only and
                 // re-enabled in the shader.
-                if (hdCamera.IsSSREnabled())
```

Comment:
why  if (hdCamera.IsSSREnabled()) have disappeard?

Reply:
As discussed with Julien during our call, this is redundant in Rendergraph as the pass will be automatically culled.

Reply:
ok